systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaUtils.java

```diff
@@ -199,19 +199,19 @@ public static void waitForClusterStability(String namespaceName, String clusterN
             boolean kafkaSameAsLast = kafkaSnaptop.equals(kafkaPods[0]);
             boolean eoSameAsLast = eoSnapshot.equals(eoPods[0]);
             if (!zkSameAsLast) {
-                LOGGER.info("ZK Cluster not stable");
+                LOGGER.warn("ZK Cluster not stable");
             }
             if (!kafkaSameAsLast) {
-                LOGGER.info("Kafka Cluster not stable");
+                LOGGER.warn("Kafka Cluster not stable");
             }
             if (!eoSameAsLast) {
-                LOGGER.info("EO not stable");
+                LOGGER.warn("EO not stable");
             }
             if (zkSameAsLast
                     && kafkaSameAsLast
                     && eoSameAsLast) {
                 int c = count[0]++;
-                LOGGER.info("All stable for {} polls", c);
+                LOGGER.debug("All stable after {} polls", c);
```

Comment:
I think you should add some final info log that pods are stable for X seconds or something like that. It seems it's not in this method.

Reply:
Provided.